src/peoplemeasurement/models.py

```diff
@@ -54,6 +54,7 @@ class Servicelevel(models.Model):
 
 
 class Area(models.Model):
+    area_id = models.CharField(max_length=255, unique=True, null=True)
```

Comment:
Ik weet niet of dit nou de beste naam is voor zo'n id. Dan is er dus een groot verschil tussen `area.id` en `area_id`. Kunnen dit ook een `gid` maken. Laten we deze ook nog bespreken.